src/pages/questions/ui/QuestionsPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { NavLink, Navigate } from 'react-router-dom';
+import cl from './QuestionsPage.module.scss';
+import PhoneImg from 'shared/assets/img/phone.svg';
+import { Mark } from 'widgets/mark';
+import { Button } from 'shared/ui/button/Button';
+import { useAppSelector } from 'shared/lib/hooks/useAppSelector/useAppSelector';
+import { useAppDispatch } from 'shared/lib/hooks/useAppDispatch/useAppDispatch';
+import { fetchQuestions } from 'entities/questions';
+import { getCompletedSurvey, getMain, getResult } from 'app/providers/router/routeConfig/routes';
+import { useQuestionsCompleted } from 'shared/lib/hooks/useQuestionsCompited/useQuestionsCompited';
+
+const QuestionsPage: React.FC = () => {
+    const { list, error } = useAppSelector((state) => state.questions);
+    const answersList = useAppSelector((state) => state.answers.list);
+    
+    const [btnState, setBtnState] = useState(true);
```

Comment:
поскольку этот стейт отвечает за то, задизейблена ли кнопка, то лучше дать ему соответствующее имя
```suggestion
    const [isBtnDisabled, setIsBtnDisabled] = useState(true);
```